beamer/agent/chain.py

```diff
@@ -293,6 +292,9 @@ def process_claims(context: Context) -> None:
         if claim.claimer_winning.is_active or claim.challenger_winning.is_active:
             maybe_withdraw(claim, context)
             maybe_challenge(claim, context)
+            if _invalidation_ready_for_l1_relay(claim) or _proof_ready_for_l1_relay(request):
```

Comment:
nitpick quesiton: Why did you decide to pull this condition check out of the funciton? If you look at all the `maybe_` functions there is typically a list of condition checks, it would be more consistent to keep it inside.

Reply:
(Exception is the l1_cost check in the other PR, because one excludes the other maybe function.